legacy/securityconfig_v6/config.yml

```diff
@@ -207,7 +207,7 @@ opendistro_security:
             # Filter to search for users (currently in the whole subtree beneath userbase)
             # {0} is substituted with the username 
             usersearch: '(uid={0})'
-            # Skip users matching a user name, a wildcard or a regex pattern
+            # Skip users matching a user name, a WildcardMatcher or a regex pattern
```

Comment:
There is a large number of `wildcard` to `WildcardMatcher` renames that need to be reverted back. Please undo those changes to continue with the PR review.

Reply:
Yeah, botched up mass-renames in a hurry. Will look into the tests as well and ping you back.

Reply:
All done, AFAICT.

Reply:
@DmitryOlshansky Please double check. For example `legacy/securityconfig_v6/config.yml` should not refer to WildcardMatcher.